.github/workflows/sweep.yml

```diff
@@ -0,0 +1,26 @@
+name: Sweep Leftover Resources
+
+on:
+  # Warning: Be careful not to run this when there are in-progress acceptance
+  # tests
+  workflow_dispatch:
+  # Sweep leftover resources at night when it is less likely for there to be an
+  # in-progress acceptance test
+  schedule:
+    - cron: 0 0 * * MON-FRI # Every night at 00:00 UTC
```

Comment:
Is it possible we could run the sweeper filtered for resources specific to a particular acceptance test? Im just considering rather than having this always run as a cron job we could just have it run as part of acc test workflows after the tests have run.

Reply:
Yeah that would be nicer. It could probably be done by adding another env-var to pass in when running `make sweep`. And also use this env-var when running acceptance tests. Then both can add this to their prefix before adding/deleting. I'll take a look and see.

But shouldn't we still have the general case cron job in case it fails for whatever reason? So that eventually the next succeeding cron job will clean it up.

Reply:
> But shouldn't we still have the general case cron job in case it fails for whatever reason? So that eventually the next succeeding cron job will clean it up.

Hmm yeah i was thinking the sweep would run regardless of any error in the acceptance tests. But i guess there would always be exceptional cases for instance a workflow getting cancelled. So i think you are right we would probably need to have something like this no matter what. Maybe we could just make this cron job only run once a week on the weekend though - just trying to avoid the conflict with a concurrently running acceptance test as much as possible.

Reply:
> for instance a workflow getting cancelled

Right. Or the `DELETE` method not working for some reason at the moment in time when the test fails or succeeds in that workflow.

> Maybe we could just make this cron job only run once a week on the weekend though - just trying to avoid the conflict with a concurrently running acceptance test as much as possible.

Yeah that makes sense. I'll change it to do it on the weekend.

Also, it's not the worst case if test fails due to some in-test resource being deleted by the sweeper. Devs can trigger tests again and they should pass. But yeah trying to minimize that scenario is probably beneficial for our sanity. Note I think we have this issue with our cleanup process in zli system tests as well (for cleaning up runners)--that "catch-all" cron job runs at night and could affect a system test running at the same time.

Reply:
Fixed in 5f6862c